share/picongpu/dockerfiles/ubuntu-1604/Singularity

```diff
@@ -1,10 +1,10 @@
 Bootstrap: docker
-From: ax3l/picongpu:0.5.0
+From: ax3l/picongpu:0.5.0-rc1
 
 
 %labels
 Maintainer Axel Huebl <a.huebl@hzdr.de>
```

Comment:
@psychocoderHPC Should @ax3l stay the maintainer or should it shift to you?

Reply:
I added this to the todo list: https://github.com/ComputationalRadiationPhysics/picongpu/projects/25
I changed the version with our version changing tool, I would keep it "wrong" because I assume @ax3l 's docker image points to the dev branch. @ax3l is this correct?

Reply:
Let us wait with merging for a replay of @ax3l  to know how he handled it in the past.

Reply:
Usually the container images and `:tag` that I upload correspond to the release, while `:latest` then also points to the last release.

We do not distribute the singularity container on our own, as you can see this is just a plain old import from the Dockerfile. But Nvidia distributes a singularity container that way via NGC, which they build on their own with this.

Don't forget to shoot an e-mail to @AdamSimpson (please CC me on it) when the release is done and a new docker image is pushed to dockerhub, so he can update it on NGC (as well as the singularity container build from it). We usually do this after the release, so in a couple of weeks from now.